safe_transaction_service/history/views.py

```diff
@@ -146,21 +146,6 @@ def get(self, request, format=None):
             return Response(self._get_info(ethereum_client))
 
 
-class ERC20IndexingView(GenericAPIView):
```

Comment:
Can you rebase? and ping me to last review :) because this was removed here #1370